docs/resources/Auto_Moderation.md

```diff
@@ -64,17 +64,19 @@ Characterizes the type of content which can trigger the rule.
 
 ###### Trigger Metadata
 
 Additional data used to determine whether a rule should be triggered. Different fields are relevant based on the
 value of [trigger_type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-trigger-types).
 
-| Field          | Type                                                                                                              | Associated Trigger Types | Description                                                               | 
-| -------------- | ----------------------------------------------------------------------------------------------------------------- | ------------------------ | ------------------------------------------------------------------------- |
-| keyword_filter | array of strings *                                                                                                | KEYWORD                  | substrings which will be searched for in content                          |
-| presets        | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | KEYWORD_PRESET           | the internally pre-defined wordsets which will be searched for in content |
-| allow_list     | array of strings *                                                                                                | KEYWORD_PRESET           | substrings which will be exempt from triggering the preset trigger type   |
+| Field                 | Type                                                                                                              | Associated Trigger Types | Description                                                               | 
+| ----------------------| ----------------------------------------------------------------------------------------------------------------- | ------------------------ | ------------------------------------------------------------------------- |
+| keyword_filter        | array of strings *                                                                                                | KEYWORD                  | substrings which will be searched for in content                          |
+| presets               | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | KEYWORD_PRESET           | the internally pre-defined wordsets which will be searched for in content |
+| allow_list            | array of strings *                                                                                                | KEYWORD_PRESET           | substrings which will be exempt from triggering the preset trigger type   |
+| mention_total_limit   | int                                                                                                               | MENTION_SPAM             | total number of mentions (role & user) allowed per message (Maximum of 50)|
```

Comment:
```suggestion
| Field               | Type                                                                                                              | Associated Trigger Types | Description                                                                | 
| ------------------- | ----------------------------------------------------------------------------------------------------------------- | ------------------------ | -------------------------------------------------------------------------- |
| keyword_filter      | array of strings *                                                                                                | KEYWORD                  | substrings which will be searched for in content                           |
| presets             | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | KEYWORD_PRESET           | the internally pre-defined wordsets which will be searched for in content  |
| allow_list          | array of strings *                                                                                                | KEYWORD_PRESET           | substrings which will be exempt from triggering the preset trigger type    |
| mention_total_limit | integer                                                                                                           | MENTION_SPAM             | total number of mentions (role & user) allowed per message (Maximum of 50) |
```

Reply:
Please reffer to [Unwanted changes](https://github.com/discord/discord-api-docs/blob/main/CONTRIBUTING.md#unwanted-changes) rule 1.

Reply:
jup pls, table formatting fixes should always be suggested and fixed

Reply:
> Please reffer to [Unwanted changes](https://github.com/discord/discord-api-docs/blob/main/CONTRIBUTING.md#unwanted-changes) rule 1.

*refer